packages/suite/src/support/messages.ts

```diff
@@ -7660,4 +7660,53 @@ export default defineMessages({
         id: 'TR_COINJOIN_ROUNDS_LEFT',
         defaultMessage: '{rounds, plural, one {# round} other {# rounds}} left',
     },
+    TR_LOADING_FUNDS: {
+        id: 'TR_LOADING_FUNDS',
+        defaultMessage: 'Loading Funds...',
+    },
+    TR_DID_YOU_KNOW: {
```

Comment:
nit: something like TR_COINJOIN_FACT_TITLE would be better, translation key should not be based purely on content that could change...